packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  Root,
+  List,
+  Trigger as RadixTab,
+  Content as Panel,
+} from '@radix-ui/react-tabs';
+import { TabsProps, TabItemProps } from '../types';
+import { Flex } from '../Flex';
+import { ComponentClassNames } from '../shared/constants';
+import classNames from 'classnames';
+import { convertStylePropsToStyleObj, prefixer } from '../shared/utils';
+
+const isTabsType = (child: any): child is React.Component<TabItemProps> => {
+  return (
+    typeof child === 'object' &&
+    child.hasOwnProperty('props') &&
+    child.props.title != null &&
+    child.props.children != null
+  );
+};
+export interface Tabs {
+  (props: TabsProps): React.ReactComponentElement<any>;
+}
+
+export const Tabs: Tabs = ({
+  alignContent,
+  alignItems,
+  ariaLabel,
+  children,
+  className,
+  defaultTab = 0,
+  direction,
+  gap = '0',
+  grow,
+  justifyContent,
+  wrap,
+  ...rest
+}) => {
+  const tabs = React.Children.map(children, (child) => {
+    if (!isTabsType(child)) {
+      console.warn(
+        'Amplify UI: <Tabs> component only accepts <TabItem> as children.'
+      );
+      return null;
+    }
+
+    return child.props;
+  });
+
+  return (
+    <Root defaultValue={`${defaultTab}`}>
+      <List aria-label={ariaLabel}>
+        <Flex
+          alignContent={alignContent}
+          alignItems={alignItems}
+          className={classNames(ComponentClassNames.Tabs, className)}
+          direction={direction}
+          gap={gap}
+          justifyContent={justifyContent}
+          wrap={wrap}
+          {...rest}
+        >
+          {tabs.map(({ className, isDisabled, title, ...rest }, index) => (
+            <RadixTab
+              className={classNames(ComponentClassNames.TabItems, className)}
+              data-grow={grow}
+              disabled={isDisabled}
+              key={index}
+              style={prefixer(convertStylePropsToStyleObj(rest, {}))}
```

Comment:
I first tried using the `View` primitive to take care of this, but I ran into issues with the style props getting passed to the `RadixTab`. There's probably a better way to pass style props to each tab.